code/src/store/features/api/appApi.ts

```diff
@@ -0,0 +1,10 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+export const appApi = createApi({
+  baseQuery: fetchBaseQuery({
+    baseUrl: "http://localhost:5000/",
+    credentials: "include"
+  }),
+  reducerPath: "appApi",
```

Comment:
Maybe we can use global constants folder for storing this keys instead of hardcoding them?

Reply:
good idea, fixed